CONTRIBUTING.md

```diff
@@ -37,7 +37,7 @@ Once you have decided to contribute to Buildah by working on an issue, check our
 backlog of [open issues](https://github.com/containers/buildah/issues) looking
 for any that do not have an "In Progress" label attached to it.  Often issues
 will be assigned to someone, to be worked on at a later time.  If you have the
-time to work on the issue now add yourself as an assignee, and set the
+time to work on the issue now add yourselves as an assignee, and set the
```

Comment:
```suggestion
time to work on the issue now, add yourself as an assignee, and set the
```
Grammar nit: 
"you" is singular so "yourself" should also be singular. But we do need a comma after "now"